### PR TITLE
Uploaded some italian books and fixed a link

### PR DESCRIPTION
**Pull request by chardozard**
Added 1 italian Assembly book, 1 italian Bash book and 1 italian C book + fixed 1 italian C book link

### CODE REVIEW

File: books/free-programming-books-it.md [hunk 1]
```diff
@@ -81,18 +81,21 @@
...
+* [La guida di Bash per principianti](http://codex.altervista.org/guidabash/guidabash_1_11.pdf) - Machtelt Garrels (PDF)
+* [Programmazione della shell Bash](https://www.aquilante.net/doc/bash_programming.pdf) - Marco Liverani (PDF)
 
 
 ### C
 
 * [Guida di Beej alla Programmazione di Rete - Usando Socket Internet](http://linguaggioc.altervista.org/dl/GuidaDiBeejAllaProgrammazioneDiRete.pdf) - Brian "Beej Jorgensen" Hall, Fabrizio Pani (PDF)
 * [Il linguaggio C - Guida pratica alla programmazione](https://eineki.files.wordpress.com/2010/02/guidac.pdf) - (PDF)
-* [Linguaggio C - ANSI C](http://www.itis.pr.it/~dsacco/itis/Olimpiadi-informatica/Libri-di-testo/LinguaggioC-R&K.pdf) - B.W. Kernighan e D.M. Ritchie (PDF)
+* [Linguaggio C - ANSI C](http://www.riochierego.it/mobile/docs/terza/lab/LinguaggioC-R&K-italiano.pdf) - B.W. Kernighan e D.M. Ritchie (PDF)
```
Comment (davorpa) [inline books/free-programming-books-it.md:98]: Binary same as we can found in archive.org

https://web.archive.org/web/20180920221053/http://www.itis.pr.it:80/~dsacco/itis/Olimpiadi-informatica/Libri-di-testo/LinguaggioC-R&K.pdf

**Comment by chardozard** [comment]
I'll make a new pull request with these things fixed

**Comment by davorpa** [comment]
### :writing_hand: :nerd_face: **TIP OF THE DAY**

Not is needed open new PRs to improve them.

Work on the same branch from where open. The upstream will auto-sync this PR when changes are pushed to your fork :rocket: 

Help mantainers :pray:. The majority of times, it convert the review process in a headache :wink:

**Comment by davorpa** [comment]
Duplicate of #6827

**Comment by davorpa** [comment]
Work continues on #6816

**Pull request closed without merging**